README.md

```diff
@@ -1,6 +1,8 @@
 # Pipeline: Groovy Plugin
 
-[Wiki page](https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Groovy+Plugin)
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/workflow-cps)](https://plugins.jenkins.io/workflow-cps)
+[![Changelog](https://img.shields.io/github/v/tag/jenkinsci/workflow-cps-plugin?label=changelog)](https://github.com/jenkinsci/workflow-cps-plugin/blob/master/CHANGELOG.md)
```

Comment:
@car-roll I realized we can still use a badge for the changelog, we just point it at `CHANGELOG.md` instead of the releases page for now.